src/mode/modeHandler.ts

```diff
@@ -505,6 +505,7 @@ export class ModeHandler implements vscode.Disposable {
 
     async handleKeyEvent(key: string): Promise<Boolean> {
         if (key === "<c-r>") { key = "ctrl+r"; } // TODO - temporary hack for tests only!
+        if (key === "<ctrl-[>") { key = "<esc>"; } // TODO - temporary hack because package.json maps ctrl+[ to the escape command
```

Comment:
Actually I think we don't need this line and the corresponding test as well because `<ctrl-[>` will only be passed in by test code, right? VS Code will ensure it will translate `ctrl-[` to `vim_esc` command and we don't need test this feature in our code, what do you think?